.github/workflows/ci.yml

```diff
@@ -28,11 +28,13 @@ jobs:
       fail-fast: false
     steps:
       - name: Checkout Repo
-        uses: actions/checkout@v2
-      - name: Prepare Simulator Runtimes
-        run: Scripts/github/prepare-simulators.sh ${{ matrix.platform }}
+        uses: actions/checkout@v4
       - name: Build
         run: |
-          swift build \
-            --sdk "$(xcode-select -p)/Platforms/iPhoneOS.platform/Developer/SDKs/iPhoneOS.sdk" \
-            --triple "arm64-apple-ios13.0"
+          # select minimum supported Xcode version
+          sudo xcode-select -s /Applications/Xcode_14.3.1.app
+          # test on the minimum supported version of the github actions runner
```

Comment:
nit: the next line isn't actually running tests – it's just building.

Reply:
+1, and I think also only building the framework, not the demo app

Reply:
My change doesn't change what it builds, I'll update the comment though since we aren't running tests as the SPM package doesn't define any